libs/bilge/src/Bilge/Response.hs

```diff
@@ -42,6 +45,19 @@ getHeader' h = fromMaybe "NO_HEADER_VALUE" . getHeader h
 getCookie :: ByteString -> Response a -> Maybe Cookie
 getCookie n r = find ((n ==) . cookie_name) (destroyCookieJar $ responseCookieJar r)
 
+-- | Retrieve the value of a given cookie name from a "Set-Cookie" header on the response
+getCookieValue :: ByteString -> Response a -> Maybe ByteString
```

Comment:
`getCookie` didn't work for me at all; From the docs it seems like you need to convince it to use the cookie jar somehow? I tried setting a Cookie Jar on the request but it didn't do the trick. Maybe someone can help me with this?

Reply:
I tried that; the cookieJar itself is completely empty even though the `Set-Cookie` header is definitely there. It doesn't show up in `getCookie` either.

Reply:
no idea what's going on there either...